deploy/docker-compose/README.md

```diff
@@ -0,0 +1,261 @@
+
+1. Download the dataset from [here](https://drive.google.com/file/d/1d8Scm3q-kHTqr_-KfnXH0rPnCgKld2Iy/view?usp=sharing)
+2. Upload the dataset as described in [Uploading Data](#uploading-data)
+3. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/liver-seg.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"811620da-381f-4daa-854d-600948e67228"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+4. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
+5. Wait for the workflow to complete and reload the Orthanc study page and expect a new series to be added.
+6. To see the output from the container, run the following commands:
+   ```bash
+   $ docker container list -a | grep monai_ai_livertumor_seg_app
+   # locate the container ID and run the following
+   $ docker logs {CONTAINER ID}
+   ```
+
+#### AI Lung Segmentation MAP
+
+##### Description
+
+In this section, we will download a DICOM dataset, upload it to Orthanc and then run the **Lung Segmentation MAP** from the
+[MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk). Finally, we can expect the AI-generated segmentation result to appear in Orthanc.
+
+1. Download the data set from [here](https://drive.google.com/file/d/1IGXUgZ7NQCwsix57cdSgr-iYErevqETO/view?usp=sharing)
+2. Upload the dataset as described in [Uploading Data](#uploading-data)
+3. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/lung-seg.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"811620da-381f-4daa-854d-600948e67228"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+4. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
+5. Wait for the workflow to complete and reload the Orthanc study page and expect a new series to be added.
+6. To see the output from the container, run the following commands:
+   ```bash
+   $ docker container list -a | grep monai_ai_lung_seg_app
+   # locate the container ID and run the following
+   $ docker logs {CONTAINER ID}
+   ```
+
+## Advanced Configuration
+
+### Docker-Compose Configuration
+
+All services included in the `docker-compose` file may be customized through the default environment file, `.env`, file 
+
+*Note: Changing any IP address or port number requires an update to the applicable service configuration files.*
+
```

Comment:
So not only do users have to change IP address in .env file but also in the settings file for the listed components? We need to make this simpler?

Reply:
the IPs are for communication between the containers, user shouldn't need to modify them unless they have conflicts.

Reply:
Okay so please add a sentence that explains this to users, that this is the local subnet for container communication. When I first read this I thought it was the IP address/subnet of the host.

Reply:
updated section